pom.xml

```diff
@@ -16,25 +16,27 @@
     <description>API for document OCR</description>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>21</java.version>
 
         <included.tests>unit-test</included.tests>
 
         <apache-commons-lang.version>2.6</apache-commons-lang.version>
         <environment-reader.version>1.3.10</environment-reader.version>
-        <structured-logging.version>1.9.26</structured-logging.version>
+        <structured-logging.version>3.0.3</structured-logging.version>
+        <!-- Needed for structured-logging -->
+        <jakarta.servlet.version>6.0.0</jakarta.servlet.version>
+
         <tess4j.version>5.8.0</tess4j.version>
         <jib-maven-plugin.version>3.3.0</jib-maven-plugin.version>
 
         <!-- Spring General -->
-        <spring.boot.version>2.7.9</spring.boot.version>
+        <spring.boot.version>3.1.3</spring.boot.version>
```

Comment:
Should we use the latest version? `3.1.3` seems to contain vulnerabilities. 

Do we need to be in version 3.1.x or could we move to 3.3.x?

Reply:
thanks @rafahop I have modified it.